keywords/utils.py

```diff
@@ -200,9 +200,14 @@ def add_cbs_to_sg_config_server_field(cluster_config):
        Format of server file in sync-gateway config if there are 3 couchbase servers
        server: "http://xxx.xxx.xx.xx,xx1.xx1.x1.x1,xx2,xx2,x2,x2:8091 """
     couchbase_server_primary_node = ""
+    sg_version = get_sg_version(cluster_config)
     cbs_servers = get_cbs_servers(cluster_config)
-    for i in range(len(cbs_servers)):
+    if compare_versions(sg_version, '1.5') < 0:
```

Comment:
Can you try without using compare_version and test it sg_version < '1.5' and see if it works?